test/requests/reviews/reviewRequest-test.ts

```diff
@@ -0,0 +1,67 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../../src/app';
+import prisma from '../../../src/db/prisma.config';
+
+chai.use(chaiHttp);
+
+type ReviewPost = {
+  shelterId: number
+  cleanliness: number
+  safety: number
+  staff: number
+}
+
+type ShelterPost = {
+  name: string
+  streetAddress: string
+  state: string
+  zip: number
+  phoneNumber: string
+}
+
```

Comment:
These type declarations are now available in shelter.types.ts and review.types.ts